src/parser/jobs/mnk/modules/Fists.js

```diff
@@ -68,12 +80,63 @@ export default class Fists extends Module {
 		this.addHook('complete', this._onComplete)
 	}
 
+	normalise(events) {
+		for (let i = 0; i < events.length; i++) {
+			const event = events[i]
+			const abilityId = event.ability.guid
+
+			// Ignore any non-ability events
+			if (!abilityId) {
+				continue
+			}
+
+			// We got a remove (PrecastStatus will handle this), or an initial apply
+			if (['removebuff', 'applybuff'].includes(event.type) && STANCES.includes(abilityId)) {
+				break
+			}
+
+			// Check for any specific casts that imply stance
+			if (event.type === 'cast') {
+				// They dun goofed
+				if (abilityId === ACTIONS.SHOULDER_TACKLE.id) {
+					break
+				}
+
+				// It was a legit Tackle, we know what's up
+				if (Object.keys(STANCE_MAP).map(Number).includes(abilityId)) {
+					events.splice(0, 0, {
+						...event,
+						ability: {
+							guid: STANCE_MAP[abilityId],
```

Comment:
not blocking; this will remove other props such as the name and icon - it's not an issue in most cases, as the `guid` is used as the primary key for pretty much every use-case, but it may cause display issues down the road - just keep it in mind if you start seeing cure icons pop up or something hah